index.html

```diff
@@ -0,0 +1,177 @@
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="about-myself">
+            <div class="myself">
+                <p class="about-title">About <br> Myself</p>
+                <p class="des">Hello I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="Right contact">
+                <a class="media-text">LET’S CONNECT</a>
+
+                <ul class="right-icons">
+                    <li><a href="https://twitter.com/Ernestbore3" target="_blank" class="media-link" rel="noopener"><i
+                                class="fab fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/ernest-bore-3b6042211/" target="_blank" class="media-link"
+                            rel="noopener"><i class="fab fa-linkedin"></i></a></li>
+                    <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i
+                                class="fab fa-medium-m"></i></a></li>
+                    <li><a href="https://github.com/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                                class="fab fa-github"></i></a></li>
+                    <li><a href="https://angel.co/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                                class="fab fa-angellist"></i></a></li>
+                </ul>
+            </div>
+            <button class="resume">Get my resume</button>
+            <div>
+                <div class="language">Languages
+                    <img src="./assets/Microverse-Student-Project-2/Normal Button/Tertiary/Disabled.png">
+                </div>
+            </div>
+            <div class="language-list">
+                <ul class="lang-list">
+                    <li>
+                        <img src="./assets/Ellipse 1.svg">
+                        <span>Javascript</span>
+                    </li>
+                    <li>
+                        <img src="./assets/Ellipse 2.svg"><span>HTML</span>
+                    </li>
+
+                    <li>
+                        <img src="./assets/Ellipse 3.svg"> <span>CSS</span>
+                    </li>
+                </ul>
+            </div>
+            <div class="framework">Frameworks
+                <img src="./assets/Normal Button/Tertiary/Icons/Icon - Chevron - Right - Larger.svg">
+            </div>
+            <div class="line"></div>
+            <div class="framework">Skills
+                <img src="./assets/Normal Button/Tertiary/Icons/Icon - Chevron - Right - Larger.svg">
+            </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`, kindly fix it